subprojects/docs/src/docs/userguide/dependencyManagementWorkingWithDependencies.adoc

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[working_with_dependencies]]
+== Working with dependencies
+
+For the examples below we have the following dependencies setup:
```

Comment:
Where does "Working with Dependencies" fit into your [IA plan](https://docs.google.com/document/d/1op4_f_TC3ESJO7xvtvu0mrLpucTMJD8uVINqZMHVfdI/edit#heading=h.qbe1l85zx8r7)?

Reply:
That's the chapter I mentioned that we need to rewrite completely. I'd probably put it under "Inspecting Dependencies" depending on the content. At first I was tempted to just delete it but there're aspects that people want to know about. I will create a new issue for it.

Reply:
Okay. This definitely needs some love.

Reply:
The content we have here right now needs to be completely scrapped from my perspective.